### PR TITLE
Add galeodes

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/galeodes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - galeodes
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/qeeqbox/galeodes
+  summary: Browsers options
+  license: AGPL-3.0-or-later
```
Comment (BastianZim) [inline recipes/galeodes/meta.yaml:38]: Are you sure that it's `or-later`?

File: recipes/galeodes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/galeodes-{{ version }}.tar.gz
+  sha256: 92465cb8485165c2889f90d708617b1f505ba484859836cf6e19f9545dcbe97d
+
+build:
+  noarch: python
```
Comment (BastianZim) [inline recipes/galeodes/meta.yaml:13]: Upstream uses `scripts` so this cannot be `noarch`. You can either patch it here (Preferred) or change it.

https://github.com/qeeqbox/galeodes/blob/90ca7192ec173749a75334d92615dade357e3ea7/setup.py#L19

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/galeodes/meta.yaml (hunk 1 above)
Comment (jan-janssen) [reply recipes/galeodes/meta.yaml:43]: https://github.com/qeeqbox/galeodes/blob/main/LICENSE#L635
```
    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU Affero General Public License as published
    by the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.
```

**Pull request merged**